src/Gateway/Events/GuildMembersChunk.php

```diff
@@ -21,7 +24,7 @@ class GuildMembersChunk
     public array $not_found;
 
     /**
-     * @var \Ragnarok\Fenrir\Parts\Presence[]
+     * @var Presence[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)